src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -40,27 +40,115 @@ object Homework :
 
   object `Boolean Operators` :
 
-    def not(b: Boolean): Boolean = ??? // here is my greatest solution
+    def not(b: Boolean): Boolean =
+      if (b == true)
+      then false
+      else true  // here is my greatest solution
 
-    def and(left: Boolean, right: Boolean): Boolean = ???
+    def and(left: Boolean, right: Boolean): Boolean =
```

Comment:
Clean solution from the first attempt? Have you stolen it?

Reply:
I did it on the first try because you gave me some hints